dotnet-desktop-guide/net/winforms/whats-new/net80.md

```diff
@@ -0,0 +1,95 @@
+---
+title: What's new in Windows Forms .NET 8
+description: Learn about what's new in Windows Forms for .NET 8. Windows Forms. .NET provides new features and enhancements over .NET 7.
+ms.date: 11/29/2023
+ms.topic: conceptual
+---
+
+# What's new for .NET 8 (Windows Forms .NET)
+
+This article describes some of the new Windows Forms features and enhancements in .NET 8.
+
+There are a few breaking changes you should be aware of when migrating from .NET Framework to .NET 8. For more information, see [Breaking changes in Windows Forms](/dotnet/core/compatibility/winforms).
+
+## New default font
+
+The default font is now the same as your Windows default font. This might affect the control layout used in your forms and controls. Previous versions of Windows Forms for .NET changed the default font to **Segoe UI, 9pt**. Windows Forms for .NET Framework uses **Microsoft Sans Serif, 8.25pt** as the default font.
+
+If the font change affects the layout of your forms, and you don't want to adjust them, you can revert to either the older default .NET font or the default .NET Framework font by setting the `<ApplicationDefaultFont>` property in your project file:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- other settings -->
+
+  <PropertyGroup>
+    <ApplicationDefaultFont>Microsoft Sans Serif, 8.25pt</ApplicationDefaultFont>
+  </PropertyGroup>
+
+</Project>
+```
+
+The Visual Studio designer respects the application's default font when set in the project file.
```

Comment:
👀 Is this a stray copy from .NET Core 3.x? Or has been a new default font added?
/cc: @merriemcgaw @dreddy-work

Reply:
This was in early releases - I wonder if @adegeo got it from the highlights video I did for .NET Conf this year.

Reply:
Yes the video was hard to distinguish what is NEW vs what is different from Framework. So this sounds like there haven't been any other changes to font to .NET 8?

Reply:
I haven't followed Windows Forms .NET 8 development train, so @merriemcgaw and @dreddy-work to confirm.
The mentioned here change was introduced in .NET Core 3.0 and it's documented [here](https://learn.microsoft.com/en-us/dotnet/core/compatibility/winforms#default-control-font-changed-to-segoe-ui-9-pt).